build_tools/version.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#/usr/bin/env bash
```

Comment:
`(error) Use #!, not just #, for the shebang. See https://github.com/koalaman/shellcheck/wiki/SC1113`
Our internal tests caught this.

Reply:
@sagar0 Of course! Sorry a small typo. I have now fixed it.